Temporary patch to make USE_PDF_PERMISSIONS section build.

For upstream.

--- a/pdf/ghostpdf.c
+++ b/pdf/ghostpdf.c
@@ -1248,6 +1248,8 @@ pdfi_device_set_flags(ctx);
     if (ctx->Trailer) {
         /* See comment in pdfi_read_Root() (pdf_doc.c) for details */
         pdf_dict *d = ctx->Trailer;
+	double Permissions = 0;
+	uint32_t P = 0;
 
         pdfi_countup(d);
         code = pdfi_dict_get(ctx, d, "Encrypt", &o);
